app/resources/css-default.template

```diff
@@ -216,6 +217,7 @@ input[type="text"] {
 input:not([type]):focus,  /* type attribute is not specified */
 input[type="text"]:focus {
   border: 0;
+  border-radius: 0;
```

Comment:
there is no need to repeat properties for x and x:focus, it should inherit them. In :focus add only what changes